data/test/vtgate/select_cases.txt

```diff
@@ -543,3 +543,30 @@
     "Values": 5
   }
 }
+
```

Comment:
Need to add tests for unsharded routes:
* Regular union
* Union in a from clause subquery
* Union in a where clause subquery